src/pages/ApplicationsPage.tsx

```diff
@@ -30,23 +42,7 @@ const ApplicationsPage = () => {
           )}
         </React.Fragment>
       ) : (
-        <React.Fragment>
-          <Helmet>
```

Comment:
This could still be here just to be consistent. Just moving the paglayout stuff would work.

Reply:
Fixed.